modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/styles/_variables.scss

```diff
@@ -12,6 +12,8 @@ $info-primary: #2e5aac;
 
 $error-l1: lighten($danger-primary, 28%);
 $gray-600: #6b6c7e;
+$gray-100: #f7f8f9;
```

Comment:
We can do `@import 'atlas-variables';` at the top of this file and have access to Clay CSS variables. We won't have to redefine them every time one is needed, although it makes it a pain to figure out what the hex value of `$gray-100` is.

Reply:
Hi @pat270! Thanks a lot for the review :) 
I did a little fix on our static env configuration in order to have `atlas-variables` working also there